package.json

```diff
@@ -41,7 +41,7 @@
     "underscore": "^1.9.1"
   },
   "engines": {
-    "node": "10.21.0",
+    "node": "^16",
     "npm": "5.3.0"
```

Comment:
I'm going to push a commit to take care of it here but please be sure to update the npm version as well in repos that already list one here. It looks like companion npm version to node 16.15.1 is 8.11.0